doc/source/Resources/Contributing.rst

```diff
@@ -135,18 +135,24 @@ Documentation
 Documentation for PyAEDT is generated from three sources:
 
 - Docstrings from the classes, functions, and modules of PyAEDT using `sphinx.ext.autodoc`.
-- reStructuredText (RST) files from `docs/`
+- reStructuredText (RST) files from `doc/`
 - Examples from `examples/`
 
-General usage and API descriptions should be placed within `docs/source` and method docstrings. Full examples should be placed in `examples/`.
+General usage and API descriptions should be placed within `doc/source` and method docstrings. Full examples should be placed in `examples/`.
```

Comment:
```suggestion
General usage and API descriptions should be placed within ``doc/source`` and method docstrings. Full examples should be placed in `examples/`.
```

Reply:
Issue with these lines is we've converted it from markdown but forgot to convert the backquotes.